packages/kbn-test/src/jest/setup/polyfills.js

```diff
@@ -18,3 +18,5 @@ if (!global.URL.hasOwnProperty('createObjectURL')) {
 // Will be replaced with a better solution in EUI
 // https://github.com/elastic/eui/issues/3713
 global._isJest = true;
+
+global.ResizeObserver = require('resize-observer-polyfill');
```

Comment:
It looks like we typically use `resize-observer` via `import { ResizeObserver } from 'resize-observer-polyfill'` and not via the global. We don't currently do something like this in the browser, and if this polyfill is necessary for `@elastic/charts` we should probably include the polyfill in `packages/kbn-ui-shared-deps-src/src/polyfills.js`

Reply:
Hey @spalger actually the ResizeObserver is supported by [all major browsers we support](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) and we don't need anymore the polyfill.
That's why I've removed the polyfill from the `@elastic/charts` package.

The problem is that the `ResizeObserver` object is not available in the `window` object of JSDOM when running the Jest test. Looks like it is not implemented in JSDOM@26. 

I don't think we need to include it into `packages/kbn-ui-shared-deps-src/src/polyfills.js` and I don't think I need to revert our changes in charts to bring back that polyfill. The polyfill should just live in the test environment because is not supported, but we should not include it in our distribution. What do you think?

Reply:
I think I've found the issue: in TS the `ResizeObserver` object is available as a global and it got compiled correctly.
Instead, in JSDOM it is not global but is only a `window` object.
I've changed the chart compiled code to use `window.ResizeObserver` instead and the tests are passing. I will push an update into charts soon to fix that

Reply:
Nice, thanks!

Reply:
Actually, this doesn't solve the problem, I can't figure it ATM so I'm reverting back our commit in the elastic-charts and to include back again the polyfill.
This is anyway an issue that needs to be solved: JSDOM doesn't mock/support the ResizeObserver API, Kibana is using 3 years old with no commits [resize-observer-polyfill](https://github.com/que-etc/resize-observer-polyfill) library.